### ISSUE TITLE
Error while translating: TypeError: object of type 'NoneType' has no len()

### ISSUE THREAD

**Issue by ewelinabuturla** [wagtail/wagtail-localize#487]
Hello, 
I've faced an issue when I tried to translate a page, specifically:

```
File "/app/.heroku/venv/lib/python3.6/site-packages/wagtail_localize/segments/extract.py", line 183, in extract_segments
    template, strings = extract_strings(field.value_from_object(instance))
File "/app/.heroku/venv/lib/python3.6/site-packages/wagtail_localize/strings.py", line 297, in extract_strings
    soup = BeautifulSoup(html, "html.parser")
File "/app/.heroku/venv/lib/python3.6/site-packages/bs4/__init__.py", line 310, in __init__
    elif len(markup) <= 256 and (
TypeError: object of type 'NoneType' has no len()`
```

When I edited the page and added a value to some random field, published and tried to translate one again - it worked. 
But the issue is definitely something worth to check.

**Comment by zerolab** [comment]
Hey @ewelinabuturla,

Thank you for the report. Which wagtail-localize version are you using?

> I've faced an issue when I tried to translate a page [...]

Did the page have content when you tried to translate it the first time?

**Comment by ewelinabuturla** [comment]
@zerolab version is 0.9.5.
yes, the page had some content the first time, but some of the fields were empty.

**Comment by zerolab** [comment]
Somewhat related: #523

**Comment by zerolab** [comment]
There's #523 which @vladox opened a while ago, but it needs work.
Will gladly review a completed PR, but I do not have time for active developement, at least not in the short term

**Comment by eathtespagheti** [comment]
I'm getting this issue too, the workaround of editing a random field doesn't work in my situation

**Comment by zerolab** [comment]
This is fixed and will make it in the 1.3 release, scheduled for the next week.

Thanks to @vladox and @Timusan for input
